R/parsed.R

```diff
@@ -1,3 +1,15 @@
+#' Parse and pre-process character vector
+#'
+#' @description The function obtains detailed parse information for `text` via
+#'   [utils::getParseData] and does some minimal pre-processing by calling
+#'   [add_ws_to_parse_data].
+#' @inheritSection add_ws_to_parse_data Details
```

Comment:
When in doubt, leave two separate functions, provided the name (which should uniquely describe the function's purpose) fits one line horizontally ;-) When splitting or refactoring functions, I'd suggest to think about what a good interface for unit-testing them would look like.